test/makefile

```diff
@@ -3,30 +3,35 @@ OBJ_DIR = obj
 TARGET = unittest
 
 CXX = g++
-CXXFLAGS = -std=c++20 -Wall -g -MMD -MP -I/usr/local/include -I../src
+CXXFLAGS = -std=c++20 -Wall -g -MMD -MP -I../src
 Catch2_CFLAGS = $(shell pkg-config --cflags catch2)
 Catch2_LDFLAGS = $(shell pkg-config --libs catch2)
 
 TEST_SRCS = $(wildcard $(SRC_DIR)/*.cpp)
-
 SRC_FILES = ../src/bmp.cpp ../src/pixelbasher.cpp
 
 SRCS = $(TEST_SRCS) $(SRC_FILES)
 
 OBJS = $(patsubst %.cpp, $(OBJ_DIR)/%.o, $(notdir $(SRCS)))
 
-$(OBJ_DIR)/%.o:
+# Rule for cpp files in current directory
+$(OBJ_DIR)/%.o: %.cpp
+	mkdir -p $(OBJ_DIR)
+	$(CXX) $(CXXFLAGS) $(Catch2_CFLAGS) -c $< -o $@
+
+# Rule for cpp files in src directory
+$(OBJ_DIR)/%.o: ../src/%.cpp
 	mkdir -p $(OBJ_DIR)
-	$(CXX) $(CXXFLAGS) $(Catch2_CFLAGS) -c $(filter %/$*.cpp, $(SRCS)) -o $@
+	$(CXX) $(CXXFLAGS) $(Catch2_CFLAGS) -c $< -o $@
 
 $(TARGET): $(OBJS)
 	$(CXX) $(CXXFLAGS) $(Catch2_CFLAGS) $(OBJS) -o $(TARGET) $(Catch2_LDFLAGS)
 
--include $(OBJS:.o=.d)
+-include $(wildcard $(OBJS:.o=.d))
```

Comment:
Is this use of `wildcard` really needed?  It looks somewhat hackish/too-clever, as it doesn't actually involve any patterns with wildcards...